patch/kconfig-inclusions

```diff
@@ -81,6 +81,92 @@ CONFIG_SENSORS_EMC2305=m
 CONFIG_SENSORS_JC42=m
 CONFIG_SENSORS_POWR1220=m
 
+# For Cisco 8000
+CONFIG_PHYLIB=m
+CONFIG_GPIOLIB=y
+CONFIG_OF_GPIO=y
+CONFIG_OF=y
+CONFIG_OF_MDIO=m
+CONFIG_MDIO_BUS_MUX_GPIO=m
+CONFIG_MDIO_BUS_MUX=m
+CONFIG_MFD_SYSCON=y
+CONFIG_REGMAP_MMIO=y
+CONFIG_ARCH_NR_GPIO=10240
+CONFIG_AD7266=m
+CONFIG_AD7291=m
+CONFIG_AD7298=m
+CONFIG_AD7476=m
+CONFIG_AD7766=m
+CONFIG_AD7791=m
+CONFIG_AD7793=m
+CONFIG_AD7887=m
+CONFIG_AD7923=m
+CONFIG_AD799X=m
+CONFIG_AXP20X_ADC=m
+CONFIG_AXP288_ADC=m
+CONFIG_CC10001_ADC=m
+CONFIG_HI8435=m
+CONFIG_HX711=m
+CONFIG_INA2XX_ADC=m
+CONFIG_MAX1027=m
+CONFIG_MAX11100=m
+CONFIG_MAX1118=m
+CONFIG_MAX1363=m
+CONFIG_MAX9611=m
+CONFIG_MCP320X=m
+CONFIG_MCP3422=m
+CONFIG_NAU7802=m
+CONFIG_TI_ADC081C=m
+CONFIG_TI_ADC0832=m
+CONFIG_TI_ADC084S021=m
+CONFIG_TI_ADC12138=m
+CONFIG_TI_ADC108S102=m
+CONFIG_TI_ADC128S052=m
+CONFIG_TI_ADC161S626=m
+CONFIG_TI_ADS1015=m
+CONFIG_TI_ADS7950=m
+CONFIG_TI_TLC4541=m
+CONFIG_SENSORS_TMP421=m
+CONFIG_SENSORS_TPS53679=m
+CONFIG_SENSORS_LTC2978=m
+CONFIG_DS4424=m
+CONFIG_SENSORS_LM75=m
+CONFIG_LTC2497=m
+CONFIG_SENSORS_LTC4151=m
+CONFIG_SENSORS_LTC2945=m
+CONFIG_SENSORS_LM25066=m
+CONFIG_SENSORS_ADM9240=m
+CONFIG_SENSORS_MAX6697=m
+CONFIG_SENSORS_LM75=m
```

Comment:
It’s already listed in line 133.

Reply:
I will close this PR and open a new one with rebase.